src/fights.ts

```diff
@@ -1957,3 +1958,32 @@ function voidMonster(): void {
   adventureMacro(determineDraggableZoneAndEnsureAccess(), Macro.basicCombat());
   postCombatActions();
 }
+
+export function printEmbezzlerLog(): void {
+  if (resetDailyPreference("garboEmbezzlerDate")) {
+    property.set("garboEmbezzlerCount", 0);
+    property.set("garboEmbezzlerSources", "");
+  }
+  const totalEmbezzlers =
+    property.getNumber("garboEmbezzlerCount", 0) +
+    embezzlerLog.initialEmbezzlersFought +
+    embezzlerLog.digitizedEmbezzlersFought;
+
+  const allEmbezzlerSources = property
+    .getString("garboEmbezzlerSources")
+    .split(",")
+    .filter((source) => source);
```

Comment:
I know what this line does but scotch would get a nosebleed and fall into an apoplectic rage upon seeing this, so it may be worth seeing if there's a slightly less funny way to filter out empty entries.

Reply:
I didn't write this lol I just moved it